feature/system/src/main/java/io/github/droidkaigi/confsched2020/system/ui/viewmodel/SystemViewModel.kt

```diff
@@ -35,4 +36,20 @@ class SystemViewModel @Inject constructor(
         // launch a Custom Tabs Activity
         customTabsIntent.launchUrl(activity, Uri.parse(url))
     }
+
+
+    fun openCalendar(
```

Comment:
The aim is not to open a calendar but saving a plan to a calendar, right? then `saveEventToCalendar` would be nicer.

Reply:
updated with this issue d4786eb
I thought `saveEventToGoogleCalendar` is better.

Reply:
I'm just curious so let me ask a question on the new name. Does it mean this code work with only Google calendar app?

Reply:
Oops, I thought it was explicit intent but wasn't. I'll remove `Google` .